modules/nf-core/meryl/unionsum/main.nf

```diff
@@ -22,6 +23,7 @@ process MERYL_UNIONSUM {
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
     meryl union-sum \\
+        k=$kvalue \\
```

Comment:
is this a mandatory arg? If not, shouldn't it be in `ext.args` ?

Reply:
Yes, mandatory otherwise errors. That's why it was moved here. I wrote some of these modules before we made that a rule.